api/desecapi/management/commands/stop-abuse.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management import BaseCommand
+
+from desecapi.models import RRset, Domain, User
+from desecapi.pdns_change_tracker import PDNSChangeTracker
+
+
+class Command(BaseCommand):
+    help = 'Removes all DNS records from the given domain and suspends the associated user'
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain-name', nargs='*', help='Domain(s) to remove all DNS records from')
+
+    def handle(self, *args, **options):
+        with PDNSChangeTracker():
+            domain_f = Domain.objects.filter(name__in=options['domain-name'])
+            user_f = User.objects.filter(domains__name__in=options['domain-name'])
+            rrset_f = RRset.objects.filter(domain__name__in=options['domain-name']).exclude(type='NS', subname='')
+            print(f'Deleting {rrset_f.count()} RRset(s) from {domain_f.count()} domain(s); '
+                  f'disabling {user_f.count()} associated user account(s).')
+            rrset_f.delete()
```

Comment:
This needs to be done by domain, as pdns (change tracker) doesn't support multi-domain updates, right?

Reply:
PDNSChangeTracker supports updates across multiple domains, but without guaranteeing atomic updates (multiple pdns API requests are used). Check [this loop](https://github.com/desec-io/desec-stack/blob/main/api/desecapi/pdns_change_tracker.py#L289). I'm unsure why the loop is only over domain additions and RRset creations; will look into further.

Reply:
PDNSChangeTracker gurantees that the key sets of additions, modifications, and deletions are always equal, so the loop is indeed over all additions, modifications, and deletions, even though it looks like it's only over additions.